File: usedesk_sdk/src/main/java/ru/usedesk/sdk/external/service/notifications/view/UsedeskSimpleNotificationsService.java
package ru.usedesk.sdk.external.service.notifications.view;

import android.app.Notification;
import android.support.annotation.NonNull;

public abstract class UsedeskSimpleNotificationsService extends UsedeskNotificationsService {
    @Override
    protected void showNotification(@NonNull Notification notification) {
        getNotificationManager().notify(5, notification);
    }
}

File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/external/service/notifications/view/UsedeskSimpleNotificationsService.kt
package ru.usedesk.chat_sdk.external.service.notifications.view

import android.app.Notification

abstract class UsedeskSimpleNotificationsService : UsedeskNotificationsService() {
    override fun showNotification(notification: Notification) {
        notificationManager.notify(5, notification)
    }
}